operators/pkg/forge/resourcequota.go

```diff
@@ -36,6 +36,12 @@ var (
 
 	// CapMemory -> The cap amount of RAM memory that can be requested by a Tenant.
 	CapMemory = *resource.NewScaledQuantity(50, resource.Giga)
+
+	// SbCapCPU -> The cap amount of CPU cores that can be requested by a Sandbox.
+	SbCapCPU = *resource.NewQuantity(8, resource.DecimalSI)
```

Comment:
```suggestion
	// SandboxCPUQuota -> The maximum amount of CPU cores that can be used by sandbox namespaces.
	SandboxCPUQuota = *resource.NewQuantity(4, resource.DecimalSI)
```

These variables have a different meaning with respect to the others.